internal/indexmanagement/scripts.go

```diff
@@ -131,6 +131,45 @@ def updateWriteIndex(currentIndex, nextIndex, alias):
     print(e)
     sys.stdout = original_stdout
     return False
+
+def getCurrentDiskPercent():
+  original_stdout = sys.stdout
+  try:
+    es_client = getEsClient()
+    response = es_client.cat.allocation(format="JSON", h="disk.percent,node", v=True)
+    diskPercent = []
+    for i in range(len(response)):
+      diskPercent.append(int(response[i]["disk.percent"]))
+    currentDiskPercent = max(diskPercent)
+    return currentDiskPercent
+  except Exception as e:
+    sys.stdout = open('/tmp/response.txt', 'w')
+    print(e)
+    sys.stdout = original_stdout
+    return False
+
+def deleteByPercentage(alias):
+  original_stdout = sys.stdout
+  try:
+    es_client = getEsClient()
+    indices = es_client.cat.indices(index=alias+"*",format="json",h="index,creation.date")
+
+    currentDiskPercent = getCurrentDiskPercent()
+    diskThreshold = os.environ['DISK_THRESHOLD']
+    
+    while (int(currentDiskPercent) >= int(diskThreshold)):
+      print ("disk space threshold is exceeded, deleting the oldest index")
+      indices = es_client.cat.indices(index=alias+"*",format="json",h="index,creation.date")
+      if not indices:
+        break
+      oldest_index = min(indices, key=lambda s: s["creation.date"])
+      response = es_client.indices.delete(index=oldest_index['index'])
+      currentDiskPercent = getCurrentDiskPercent()
+  except Exception as e:
+    sys.stdout = open('/tmp/response.txt', 'w')
+    print(e)
+    sys.stdout = original_stdout
+
```

Comment:
Do we want to return True or False over here?

Reply:
Yes you're right.